elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -45,13 +73,32 @@ def render(element_html, data):
                 prefix = 'np.array('
                 suffix = ')'
 
+            # Mathematica reserved letters: C D E I K N O
+            mathematica_reserved = ['C', 'D', 'E', 'I', 'K', 'N', 'O']
+            if pl.inner_html(child) in mathematica_reserved:
+                mathematica_suffix = 'm'
+            else:
+                mathematica_suffix = ''
+
             # Create string for matlab and python format
-            matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=digits) + ';\n'
-            python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=digits) + suffix + '\n'
+            if pl.has_attrib(child, 'params-comment'):
+                matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=var_digits) + '; % ' + var_comment + '\n'
```

Comment:
can we just set variables like `var_name = pl.inner_html(child)`, `var_formatted_data = pl.string_from_2darray(var_data, language=var_language, digits=var_digits)` etc, so that we don't have to repeat all this code?

Reply:
Couldn't think of how to make `var_language` work with the current implementation. I made separate `var_language_data` which still cleans up some of the repeated code